src/TypeScript/BuildTypeScript.ps1

```diff
@@ -0,0 +1,7 @@
+
+# TODO:  Expand on this.
+cd Sarif.Sdk
```

Comment:
you will find pushd/popd is preferable for this sort of thing, to avoid need to update relative paths in a complex script when directory nesting changes. just an historical maintainability suggestion for batch files. :)